core/frontend/src/render/RenderSystem.ts

```diff
@@ -440,6 +440,9 @@ export abstract class RenderSystem implements IDisposable {
     });
   }
 
+  /** Create a new texture by its name. This texture will be retrieved asynchronously from the backend. A placeholder image will be associated with the texture until the requested image data loads. */
+  public createTextureFromExternalImage(_name: string, _imodel: IModelConnection, _params: RenderTexture.Params, _format: ImageSourceFormat): RenderTexture | undefined { return undefined; }

```

Comment:
nitpick: most places where you're using the term "external texture" you really mean "a texture element persisted in the iModel." "External" sounds like it could come from anywhere, e.g., an arbitrary URL.

Reply:
I see what you mean.  Any suggestions for another phrase, particularly something pithy enough to go in a method name like this?

Reply:
`RenderSystem.createTextureFromElement(elementId: Id64String, ...)`?

Reply:
I changed this phrasing in many of the methods to be *FromElement(id: Id64String, ...)